src/main/java/org/apache/sysds/runtime/privacy/finegrained/FineGrainedPrivacyMap.java

```diff
@@ -40,6 +40,18 @@ public void put(DataRange dataRange, PrivacyLevel privacyLevel) {
 		constraintCollection.put(dataRange, privacyLevel);
 	}
 
+	@Override public void putRow(int rowIndex, int rowLength, PrivacyLevel privacyLevel) {
```

Comment:
misformatted, and does nothing inside the method.

Reply:
I know. I am not using that class and will perhaps delete it, but I wanted to get the propagation done and then see how that performs with the other implementation. Then I can try different implementations of FineGrainedPrivacy.

Reply:
if it is not used, delete it :+1: